content/docs/user-guide/how-to/stop-tracking-data.md

```diff
@@ -8,9 +8,7 @@ description: 'You can "un-track" files or directories added in error.'
 There are situations where you may want to "un-track" files or directories added
 in error to DVC.
 
-<details>
-
-## Expand to add a sample data `data.csv` file
+#Expand to add a sample data `data.csv` file
```

Comment:
Does removing the `<details>` here break something?

Reply:
Oh weird, this is not the change I intended ⏳